labeling/labels/labels.go

```diff
@@ -53,11 +56,9 @@ func (label Label) String() string {
 type LabelSet map[string]Label
 
 func (ls LabelSet) String() string {
-	labelsAsStrings := make([]string, len(ls))
-	i := 0
+	labelsAsStrings := make([]string, 0, len(ls))
 	for _, v := range ls {
-		labelsAsStrings[i] = v.String()
-		i++
+		labelsAsStrings = append(labelsAsStrings, v.String())
```

Comment:
Curious why this is preferred? Seems like more allocations (but unlikely to make significant diff).

Reply:
I forgot to revert this change, this is when I wanted to avoid adding the cicd to the stack comment. Because of the pre-allocation I was getting a bunch of empty strings. I will revert this change.